h/models/annotation.py

```diff
@@ -189,6 +192,10 @@ def target_uri_normalized(self):
     @property
     def target(self):
         target = {"source": self.target_uri}
+
+        if self.target_description is not None:
+            target["description"] = self.target_description
```

Comment:
If `self.target_description` is `None` then don't return any `"description"` in the `target` dict at all. This ends up resulting in the API returning omitting the `target.description` field when returning annotations. (I'm not in love with the shape of API responses being controlled by the SQLAlchemy model code like this, but it's inline with the existing pattern for `target.source` above.)